internal/common/error_utils.go

```diff
@@ -14,6 +14,8 @@ import (
 	"gorm.io/gorm"
 )
 
+const APINotFound = "V1 API not found"
```

Comment:
This needs to be rebased on the other PR?

Reply:
Done.